vcr/holder/sql_wallet.go

```diff
@@ -113,7 +113,23 @@ func (h sqlWallet) Put(_ context.Context, credentials ...vc.VerifiableCredential
 }
 
 func (h sqlWallet) List(_ context.Context, holderDID did.DID) ([]vc.VerifiableCredential, error) {
-	return h.walletStore.list(holderDID)
+	credentials, err := h.walletStore.list(holderDID)
+	if err != nil {
+		return nil, err
+	}
+	// now validate credentials and remove invalid ones
```

Comment:
The `NutsAuthorizationCredential` is still heavily in use meaning that `credentials` is going to contain a lot of invalid credentials. Can we filter this during the SQL query?

Reply:
I think they are not in the SQL wallet?

Reply:
`vcr.go#canWalletHoldCredential` filters NutsAuthCreds so they won't go to the wallet. @gerardsn